src/main/interface/settings.c

```diff
@@ -365,7 +370,10 @@ const lookupTableEntry_t lookupTables[] = {
 const clivalue_t valueTable[] = {
 // PG_GYRO_CONFIG
     { "align_gyro",                 VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_ALIGNMENT }, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyro_align) },
-    { "gyro_lpf",                   VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_GYRO_LPF }, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyro_lpf) },
+    { "gyro_hardware_lpf",          VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_GYRO_HARDWARE_LPF }, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyro_hardware_lpf) },
+#if defined(USE_GYRO_SPI_MPU6500) || defined(USE_GYRO_SPI_MPU9250) || defined(USE_GYRO_SPI_ICM20689)
```

Comment:
I am wondering if it would make sense to use `USE_32K_CAPABLE_GYRO` or similar for this, and define it in `common_fc_post.h` if one of the supported gyros is included?

Reply:
That would probably be a good idea as a separate follow-on PR.  That 32K capable `if defined()` logic is scattered all over the place and outside the scope of this PR.  We really have two different issues:
1. Target-based determination
2. Runtime determination

The problem with 1 is that some targets have multiple gyro types available and we end up getting code and features included for 32K capable just becasue the target "could" have one of those gyros.  For example the REVO target and variants.  This ends up including code that isn't needed for non-32k capable instances.

For 2. we should start using runtime determination based on the actual gyro detected.  This would apply to targets that have multiple gyro options, and dual-gyro boards that can select different types at runtime.

Reply:
My point was more about replacing the multiple instances of `#if defined(USE_GYRO_SPI_MPU6500) || defined(USE_GYRO_SPI_MPU9250) || defined(USE_GYRO_SPI_ICM20689)` with `#if defined(USE_32K_CAPABLE_GYRO)`, to make it easier to maintain.
The over-arching cleanup of the 32 kHz code is definitely something for a new pull request.

Reply:
I get that.  However I'd like to do the `#if defined(USE_32K_CAPABLE_GYRO)` change as part of a follow-on PR.  This logic is scattered all over the place and mixing it into this PR could create problems if we have to bisect.